getting-started/exploring-cloud.md

```diff
@@ -0,0 +1,119 @@
+# Exploring Timescale Cloud
+
+Welcome to Timescale Cloud. Timescale Cloud is a Database as a Service (DBaaS) 
+offering that provides an easy way for you to analyze time-series. Powered 
+by [TimescaleDB][timescale-features], you can create database instances in the 
+cloud and automate many of your most common operational tasks. This allows you 
+to spend more time focusing on your time-series workloads and less time worrying 
+about database management.
+
+Start by [creating a Timescale Cloud account][create-login]. Once you have login 
+credentials, you can access the [Timescale Cloud portal][timescale-cloud-portal].
+
+Before we start, let's review a few core concepts and phrases:
+
+- **Account**: Your Timescale Cloud account. You can register for a Timescale Cloud account on the [Timescale Cloud signup][sign-up] page.
+- **Project**: An empty project is created for you automatically when you sign-up and are the highest organization level. Projects organize groups of services, and have different billing settings. A way to organize Services in your account and provide access to Services with other users. This is the highest level of access.
+- **Service**: Multiple services can be created under each project, and each service corresponds to a cloud service provider tier (e.g. AWS Timescale-Pro-512-IO-Optimized).
+- **Database**: Databases are created within the context of a single server, and can be viewed and created after selecting one of your services, and then selecting the databases tab.
+- **Service Plans**: A Service Plan defines the configuration and level of database management that will be performed for a given TimescaleDB deployed.
+
+### Step 1: Sign up for Timescale Cloud
```

Comment:
This step seems somewhat redundant, since the second paragraph instructs the user to sign up for a Timescale Cloud account.  Also, the fact that cloud signup is mentioned three times in succession seems a bit pushy?